Exiled.API/Features/Map.cs

```diff
@@ -179,6 +180,8 @@ public static Room FindParentRoom(GameObject objectInRoom)
                     // room = FindParentRoom(role.Camera.GameObject);
             }
 
+            room = Room.Get(objectInRoom.transform.position);
```

Comment:
This overrides the room found with `objectInRoom.GetComponentInParent<Room>();` at the line 169.